common/archiver/filestore/historyArchiver.go

```diff
@@ -236,7 +237,8 @@ func (h *historyArchiver) Get(
 		return nil, &shared.EntityNotExistsError{Message: archiver.ErrHistoryNotExist.Error()}
 	}
 
-	encodedHistoryBatches, err := readFile(filepath)
+	// #nosec
+	encodedHistoryBatches, err := ioutil.ReadFile(filepath)
```

Comment:
@andrewjdawson2016 can you review this? My guess is that indirection existed for a reason.  What's the impact of directly calling ioutil.ReadFile here?